web/src/main/kotlin/no/nav/su/se/bakover/web/routes/søknadsbehandling/SøknadsbehandlingRoutes.kt

```diff
@@ -425,6 +425,7 @@ internal fun FeilVedHentingAvGjeldendeVedtaksdataForPeriode.tilResultat(): Resul
     }
 }
 
+// TODO: egen validering for alder her? Må ha et bedre system for å skille på sakstype spesifikke feil
 internal fun Sak.KunneIkkeOppdatereStønadsperiode.tilResultat(): Resultat {
     return when (this) {
         is Sak.KunneIkkeOppdatereStønadsperiode.FantIkkeBehandling -> fantIkkeBehandling
```

Comment:
Hva gir mening å dele opp her for å ikke få `aldersvurdering_gir_ikke_rett_på_uføre` hvis det er snakk om en alderssøknad?